src/AccountManagement.ts

```diff
@@ -110,9 +112,9 @@ export class AccountManagement {
   /**
    * Send an invitation to an Account to join the signing Identity as a secondary Account
    *
-   * @note this will create an AuthorizationRequest which has to be accepted by
-   *   the corresponding Account. An Account can
-   *   fetch its pending Authorization Requests by calling `authorizations.getReceived`
+   * @note this will create an AuthorizationRequest which has to be accepted by the corresponding Account.
```

Comment:
```suggestion
   * @note this will create an AuthorizationRequest which has to be accepted by the target Account.
```

(nit) The field name is target so I think it's a bit clearer to use that word. OK to leave as is though.